lib/r10k/cli/deploy.rb

```diff
@@ -23,6 +23,8 @@ def self.command
 
         required :c, :config, 'Specify a configuration file'
 
+        required nil, :cachedir, 'Specify a cachedir, overriding the value in config'
```

Comment:
This location makes it available to all the "deploy" subcommands if they are updated to accept and pass it through, is there a place we can put the acceptance and override functionality that will just apply to everything without updating it per action?